pkg/cmd/config.go

```diff
@@ -0,0 +1,45 @@
+package cmd
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/stripe/stripe-cli/pkg/config"
+)
+
+type configCmd struct {
+	cmd    *cobra.Command
+	config *config.Config
+
+	list  bool
+	edit  bool
+	unset bool
+}
+
+func newConfigCmd() *configCmd {
+	cc := &configCmd{
+		config: &Config,
+	}
+	cc.cmd = &cobra.Command{
+		Use:   "config",
+		Short: "Manually change the config values for the CLI",
+		RunE:  cc.runConfigCmd,
+	}
+
+	cc.cmd.Flags().BoolVar(&cc.list, "list", false, "list configs")
+	cc.cmd.Flags().BoolVarP(&cc.edit, "edit", "e", false, "open editor to the config file")
+	cc.cmd.Flags().BoolVar(&cc.unset, "unset", false, "unset a specific config field")
+
+	return cc
+}
+
+func (cc *configCmd) runConfigCmd(cmd *cobra.Command, args []string) error {
+	if len(args) == 2 {
```

Comment:
Does this work even if the user specifies `--project-name` in the command?

Reply:
yep! that's handled at the profile level